### DISCUSSION TITLE
"bypassing" ops by dragging cables onto each other

### DISCUSSION THREAD

**Discussion by wayfu** [cables-gl/cables_docs#855] [Feature Requests]
it would be cool if you could drag a cable onto another of the same type and cables would then disconnect both those cables and create a new one from the out port of cable 1 to the in port of cable 2

**Comment by pandrr** [comment]
https://github.com/user-attachments/assets/10ef8401-5875-4495-8897-07fa0c691212

you can do this again on dev.cables.gl it was actually just broken

**Comment by wayfu** [comment]
hmm, i can't get it to work here. which mouse button am i supposed to use?

**Comment by wayfu** [comment]
looks like what gave me trouble understanding what's happening here was that in your video, you can RMB-grab a cable from much further away from the port it is connected to than i can at home (i need to click on the port, you can click dozens of pixels away)

**Reply by pandrr** [reply]
ah sorry i forgot,i changed my preferences to enable dragging cables with right mouse button.(and scroll to middle mouse button). this is turned off by default to reduce accidental disconnecting when trying to scroll...

<img width="576" alt="Screenshot 2024-09-20 at 10 45 39" src="https://github.com/user-attachments/assets/bd6f2ce9-0860-4890-88b6-3b289096a378">
